tests/FSharp.Core.UnitTests/FSharp.Core.UnitTests.fsproj

```diff
@@ -106,4 +107,8 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Remove="FSharp.Core\OperatorsModule2.fs" />
```

Comment:
Is this still necessary now that you  have removed:

````
<Content Include="**/*" 
Exclude="**/*.bak;Directory.Build.Props;Directory.Build.targets;FSharp.Core.UnitTests.fsproj" CopyToOutputDirectory="never" />have removed:  
````

Reply:
Sorry, I missed this. Now fixed: https://github.com/dotnet/fsharp/pull/9628